zgw_consumers/api_models/base.py

```diff
@@ -32,6 +32,7 @@ def noop(value: Any) -> Any:
     datetime: parse,
     date: date.fromisoformat,
     relativedelta: parse_relativedelta,
+    bool: lambda s: s.lower() == 'true',
```

Comment:
I expected this instead:
```suggestion
    bool: bool,
```

does the remote API return strings in JSON rather than actual literals `false`/`true`?